src/test/functional/queryServices.test/tags.integration.ts

```diff
@@ -365,6 +368,164 @@ describe('tags query tests - happy path', () => {
+        count: 2,
+      },
+      {
+        user_id: 1,
+        grouping_id: 10631689,
+        tag: 'travel',
+        weighted_count: 0.9211,
+        count: 2,
+      },
+    ]);
+
+    await db('suggested_tags_user_groupings').insert([
+      {
+        user_id: 1,
+        grouping_id: 10631688,
+        weighted_count: 1.6461,
+        count: 1,
+      },
+      {
+        user_id: 1,
+        grouping_id: 10631689,
+        weighted_count: 1.6461,
+        count: 1,
+      },
+    ]);
+
+    const variables = {
+      userId: '1',
+      itemId: '1',
+    };
+
+    const GET_SUGGESTED_TAGS_FOR_SAVED_ITEM = gql`
+      query getSavedItem($userId: ID!, $itemId: ID!) {
+        _entities(representations: { id: $userId, __typename: "User" }) {
+          ... on User {
+            savedItemById(id: $itemId) {
+              url
+              suggestedTags {
+                ... on Tag {
+                  id
+                  name
+                  _createdAt
+                  _updatedAt
+                  _version
+                  _deletedAt
+                  savedItems {
+                    edges {
+                      cursor
+                      node {
+                        url
+                        item {
+                          ... on Item {
+                            givenUrl
+                          }
+                        }
+                      }
+                    }
+                    pageInfo {
+                      startCursor
+                      endCursor
+                      hasNextPage
+                      hasPreviousPage
+                    }
+                    totalCount
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    `;
+
+    const res = await server.executeOperation({
+      query: GET_SUGGESTED_TAGS_FOR_SAVED_ITEM,
+      variables,
+    });
+    expect(res.data?._entities[0].savedItemById.url).to.equal('http://abc');
+    expect(res.data?._entities[0].savedItemById.suggestedTags[0].name).to.equal(
+      'travel'
+    );
+    const tagId0 = Buffer.from(
+      res.data?._entities[0].savedItemById.suggestedTags[0].id,
+      'base64'
+    ).toString();
+    expect(tagId0).to.equal('travel');
+    expect(res.data?._entities[0].savedItemById.suggestedTags[0]._version).is
+      .null;
+    expect(res.data?._entities[0].savedItemById.suggestedTags[0]._deletedAt).is
+      .null;
+    expect(
+      res.data?._entities[0].savedItemById.suggestedTags[0]._createdAt
+    ).to.equal(unixDate);
+    expect(
+      res.data?._entities[0].savedItemById.suggestedTags[0]._updatedAt
+    ).to.equal(unixDate1);
+    expect(
+      res.data?._entities[0].savedItemById.suggestedTags[1]._createdAt
+    ).to.equal(unixDate1);
+    expect(
+      res.data?._entities[0].savedItemById.suggestedTags[1]._updatedAt
+    ).to.equal(unixDate1);
+    expect(res.data?._entities[0].savedItemById.suggestedTags[1].name).to.equal(
```

Comment:
should these be `suggestedTags[0]`?

Reply:
I think the test ordering is confusing will re-arrange